### PR TITLE
build heroku-22

### PR DESCRIPTION
**Pull request by dzuelke**
GUS-W-10340283

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -13,9 +13,9 @@ jobs:
     parameters:
       stack-version:
         type: enum
-        enum: ["18", "20"]
+        enum: ["18", "20", "22"]
     machine:
-      image: ubuntu-2004:202107-02
+      image: ubuntu-2004:202111-02
```
Comment (edmorley) [inline .circleci/config.yml:18]: Latest tag is `202201-02`:
https://circleci.com/developer/machine/image/ubuntu-2004#image-tags

```suggestion
      image: ubuntu-2004:202201-02
```

**Review by edmorley — APPROVED** [review]
I'm curious about the massive build image size reduction?

```
       ubuntu:22.04             78MB
       heroku/heroku:22         616MB
       heroku/heroku:22-build   992MB
```

```
       ubuntu:20.04             72.8MB
       heroku/heroku:20         580MB
       heroku/heroku:20-build   1.37GB
```

File: .circleci/config.yml (hunk 1 above)
Comment (edmorley) [reply .circleci/config.yml:18]: (I presume you copied the version from https://circleci.com/docs/2.0/configuration-reference/#available-machine-images; unfortunatelty those docs are outdated, I've filed https://github.com/circleci/circleci-docs/issues/6570)

Comment (dzuelke) [reply .circleci/config.yml:18]: Ah good catch; I was taking this from https://circleci.com/docs/2.0/configuration-reference/#available-machine-images I guess is out of date.

**Pull request merged**